--- cage-0.1.5/meson.build.orig	2023-07-28 16:46:37.000000000 +0200
+++ cage-0.1.5/meson.build	2024-06-28 23:10:30.722351855 +0200
@@ -35,7 +35,7 @@
   )
 endif
 
-wlroots        = dependency('wlroots', version: '>= 0.16.0', fallback: ['wlroots', 'wlroots'])
+wlroots        = dependency('wlroots', version: ['>= 0.16.0', '< 0.17.0'], fallback: ['wlroots', 'wlroots'])
 wayland_protos = dependency('wayland-protocols', version: '>=1.14')
 wayland_server = dependency('wayland-server')
 xkbcommon      = dependency('xkbcommon')
